File: sprint3/By lessons/lesson2code1/lesson2code1.cpp
#include <iostream>

void stairs_builder(int n) {
    // build 1 step
    std::cout << "Осталось построить " << n << " ступеней." << std::endl;
    stairs_builder(n - 1);
}

int main() {
    stairs_builder(5);
    return 0;
}

File: sprint3/By lessons/lesson2code1/lesson2code1.java
class Main {
    static void stairs_builder(int n) {
        // build 1 step
        System.out.printf("Осталось построить %d ступеней.\n", n);
        stairs_builder(n - 1);
    }

    public static void main(String[] args) {
        stairs_builder(5);
    }
}
